gnovm/pkg/gnomod/pkg.go

```diff
@@ -5,40 +5,32 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 type Pkg struct {
-	name     string
-	path     string
-	draft    bool
-	requires []string
+	Dir      string   // absolute path to package dir
+	Name     string   // package name
+	Requires []string // dependencies
+	Draft    bool     // whether the package is a draft
+}
+
+type SubPkg struct {
+	Dir        string   // absolute path to package dir
+	ImportPath string   // import path of package
+	Root       string   // Root dir containing this package, i.e dir containing gno.mod file
+	Imports    []string // imports used by this package
```

Comment:
ImportPaths []Name // I think ImportPkgPath is just too long.
or
ImportDirs []string // but I don't think this is what we want.

Reply:
We already have `ImportPath string // import path of package`

Don't you think `ImportPaths []Name` will make things more confusing?

`ImportDirs []string`? No. since they are not dirs